### PR TITLE
perf(feedback): rm save_event_feedback celery task

### PR DESCRIPTION
**Pull request by aliu39**
- call create_feedback_issue synchronously, since celery can be flaky and we don't want to lose feedbacks
- this won't hurt performance of the ingest-events consumer since we're moving feedback to its own topic+consumer
- moves test to its own directory in preparation for more ingest-feedback consumer tests

### CODE REVIEW

File: src/sentry/ingest/consumer/processors.py
Comment (cmanallen) [inline src/sentry/ingest/consumer/processors.py:191]: Let's delete the save_event_feedback function.

**Review by cmanallen — CHANGES_REQUESTED** [review]

File: src/sentry/ingest/consumer/processors.py
Comment (aliu3ntry) [reply src/sentry/ingest/consumer/processors.py:191]: discussed on a call:
- wait until feedback consumer is deployed (to prod?) and actually consuming all the feedbacks, before merging this change
- same with deleting save_event_feedback

Comment (cmanallen) [reply src/sentry/ingest/consumer/processors.py:191]: You can delete save_event_feedback on this pull.  We're not going to merge it until we're ready so its cool.

Comment (anonrig) [reply src/sentry/ingest/consumer/processors.py:191]: +1

File: tests/sentry/feedback/consumer/test_processing.py
Comment (anonrig) [inline tests/sentry/feedback/consumer/test_processing.py:76]: Can we pass use_rapid_json to here?

File: tests/sentry/feedback/consumer/test_processing.py
Comment (anonrig) [inline tests/sentry/feedback/consumer/test_processing.py:22]: can we add types?

File: src/sentry/ingest/consumer/processors.py
Comment (anonrig) [inline src/sentry/ingest/consumer/processors.py:191]: Can we make this a feature flag first?

**Review by Swatinem — APPROVED** [review]

**Comment by JoshFerge** [comment]
> > call create_feedback_issue synchronously, since celery can be flaky and we don't want to lose feedbacks
> 
> What makes user feedback here special? If we have a problem with celery we need to solve this. User feedback on the list of concerns is very low down the last and if we have indications that we are losing events, then this is the wrong way to go about it from where I stand.

flaky is probably an overstatement, but in this particular case, if there is no need/benefit from celery, do we need to use it? and from what I understand, there has been a push by the streaming team / architecturally, to rely on better kafka consumer strategies as opposed to using celery. enhanced durability is a nice side effect.

File: src/sentry/ingest/consumer/processors.py
Comment (aliu3ntry) [reply src/sentry/ingest/consumer/processors.py:191]: @anonrig `save_event_feedback` (the celery task) is just a wrapper for `create_feedback_issue`, so I don't think this is necessary

**Comment by aliu3ntry** [comment]
Yeah, as Josh said, I might have worded my first comment too strongly. Nothing suggests that Celery is a significant issue for performance or reliability, this could be a "ref(feedback)" instead... Basically, we didn't see a benefit from celery for this particular usecase, so we're removing it.

**Comment by cmanallen** [comment]
All: This consumer will receive **1 message every 10 seconds**.  There is no load concern.  There is no concern with celery or its ability to handle feedback ingestion.

@JoshFerge @aliu3ntry  We don't need this PR.  Let's migrate to the new feedback consumer first.  We can write a custom consumer strategy (defined in `sentry/feedback/consumers/`) later.  Its a nice onboarding issue and its not particularly difficult.  We can drop the feedback code in the event pipeline once everything is out and running.

**Comment by cmanallen** [comment]
@aliu3ntry Did you see my previous comment?  Would you like to close this and pursue a different path or continue with this PR?

**Pull request closed without merging**
